Add a more detailed introduction for model compression (#772)
deepmodeling/deepmd-kit: doc/getting-started.md

optional arguments:
  -h, --help            show this help message and exit
  -v {DEBUG,3,INFO,2,WARNING,1,ERROR,0}, --log-level {DEBUG,3,INFO,2,WARNING,1,ERROR,0}
                        set verbosity level by string or number, 0=ERROR,
                        1=WARNING, 2=INFO and 3=DEBUG (default: INFO)
  -l LOG_PATH, --log-path LOG_PATH
                        set log file to log messages to disk, if not
                        specified, the logs will only be output to console
                        (default: None)
  -m {master,collect,workers}, --mpi-log {master,collect,workers}
                        Set the manner of logging when running with MPI.
                        'master' logs only on main process, 'collect'
                        broadcasts logs from workers to master and 'workers'
                        means each process will output its own log (default:
                        master)
  -i INPUT, --input INPUT

>                         The original frozen model, which will be compressed by

Reviewer: ```suggestion
                        The original frozen model, which will be compressed
```

>                         the deepmd-kit (default: frozen_model.pb)

Reviewer: ```suggestion
 (default: frozen_model.pb)
```

>                         the deepmd-kit (default: frozen_model.pb)

Reviewer: ```suggestion
                        The original frozen model, which will be compressed. (default: frozen_model.pb)
```

>                         the deepmd-kit (default: frozen_model.pb)

Reviewer: ```suggestion
which will be compressed (default: frozen_model.pb)
```

  -o OUTPUT, --output OUTPUT
                        The compressed model (default:
                        frozen_model_compress.pb)
  -s STEP, --step STEP  Model compression introduces two tables with different
                        uniform step size to store the parameters of the
                        fifth-order polynomials. The step parameter denotes
                        the uniform step size of the first table and the
                        second table will use 10 * step as it's uniform step

>                         size. Smaller step with higher accuracy and bigger

Reviewer: ```suggestion
                        size. Smaller step means higher accuracy and bigger
```

                        model size (default: 0.01)
  -e EXTRAPOLATE, --extrapolate EXTRAPOLATE
                        The domain range of the first table is automatically
                        detected by the code, while the second table ranges
                        from the first table's upper boundary(upper) to the
                        extrapolate(parameter) * upper. (default: 5)
  -f FREQUENCY, --frequency FREQUENCY

>                         The frequency of tabulation overflow check(If the

Reviewer: ```suggestion
                        The frequency of tabulation overflow check(Check to see whether the
```
